RADWIMPS.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const p = c => process.stdout.write(c);
+const f = Promise.prototype.then;
+
+Promise.prototype.then = function () { return f.call(this, () => p('前')); };
+Promise.prototype.世 = function () { return f.call(this, () => p('世')); };
```

Comment:
```suggestion
Promise.prototype.世 = function () { return f.call(this, () => p('世\n')); };
```

最後に改行入れたほうが良いかもしれません

Reply:
9761516